src/test/java/StorageTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
Static imports should be above other imports. (Under Package and Import Statements in coding standards)